crates/ra_hir/src/nameres/tests/mods.rs

```diff
@@ -172,6 +172,511 @@ fn module_resolution_module_with_path_non_crate_root() {
     "###);
 }
 
+#[test]
+fn module_resolution_module_decl_path_super() {
+    let map = def_map_with_crate_graph(
+        "
```

Comment:
let's use an `r#"` literal here to avoid escaping `"` and `\` everywhere?

Reply:
Fixed